http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -1242,11 +1238,12 @@ class LeonardoServiceSpec
 
     val finalQueueSize = queue.getSize.unsafeRunSync()
 
-    finalQueueSize shouldBe 1
+    finalQueueSize shouldBe 2 //one message for creating cluster, one message for updating cluster
 
-    val message = queue.dequeue1.unsafeRunSync().asInstanceOf[StopUpdateMessage]
+    queue.dequeue1.unsafeRunSync() //discard createCluster message
+    val message = queue.dequeue1.unsafeRunSync().asInstanceOf[StopUpdate]
 
-    val castMessage = message.asInstanceOf[StopUpdateMessage]
+    val castMessage = message.asInstanceOf[StopUpdate]
```

Comment:
I know it was there already, but isn't this `asInstanceOf` redundant with the one on line 1244?

Reply:
yea...this is an oversight..